src/smc-webapp/printing.coffee

```diff
@@ -356,6 +358,9 @@ class SagewsPrinter extends Printer
                         data_url = scene.static_image
                         out ?= ''
                         out += "<div class='output sage3d'><img src='#{data_url}'></div>"
+                else if ext == 'webm'
+                    out ?= ''
+                    out += "<video src='#{mesg.file.url}' class='sagews-output-video' controls></video>"
```

Comment:
I'm sure you're aware, and I agree with this -- but unlike with images, this will require internet access to work...